services/movie.py

```diff
@@ -1,13 +1,18 @@
 from django.db.models import QuerySet
 
+from django.db import transaction
+
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
```

Comment:
The variable name `genres_ids` may be a bit misleading. It seems to suggest that you are expecting multiple ids for a single genre. Consider renaming it to `genre_ids` to make it clear that it refers to ids of multiple genres.